content/i18n/en/landing-page.json

```
@@ -27,32 +28,31 @@
   "keyFeatures": {
     "headline": "Key features",
     "paragraph": "Kyma {spelling} is a platform for extending applications with serverless functions and microservices.",
-    "spelling": "/kee-ma/",
     "features": {
       "extensibility": {
-        "title": "Extensibility platform",
+        "title": "Serverless",
         "paragraphs": [
-          "If you are looking for cloud-native ways to extend your enterprise products, Kyma is a secure and tested environment you can rely on.",
-          "First, connect external APIs and ensure a stable event flow between the external or built-in services and your own applications. Then, start molding your own code into microservices and functions that extend your applications' logic."
+          "Kyma empowers developers to easily deploy workloads in form of Functions. With Kyma, you can focus on coding.",
+          "Simply write your Function code, add a configuration manifest, and push it to Kyma using supporting CLI or directly to git."
         ]
       },
       "runtime": {
-        "title": "Application runtime",
+        "title": "Eventing",
         "paragraphs": [
-          "Build standalone Kubernetes applications from scratch. Kyma is also an application runtime that is scalable, secure, and reliable. It allows you to develop microservices-based solutions in any language and run them on Kubernetes clusters from a cloud provider of your choice."
+          "With Kyma, you can trigger workloads (Functions or microservices) using in-cluster events or external business events."
         ]
       },
       "support": {
-        "title": "Operational support",
+        "title": "Observability",
         "paragraphs": [
-          "Kyma ensures smooth Day-2 support for the standalone applications and extensions you'll create. Thanks to numerous CNCF open-source projects that come preconfigured with Kyma, you can easily monitor the performance of your microservices and functions, scan logs for errors, and upgrade your solutions painlessly."
+          "Kyma comes with pre-configured open-source observability tools. You can easily monitor the performance of your microservices and Functions, scan logs for errors, and trace the request flow through the entire system."
         ]
       }
     }
   },
   "inANutshell": {
-    "headline": "In a nutshell",
-    "paragraph": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+    "headline": "In the nutshell",
```

```suggestion
    "headline": "In a nutshell",
```

"In a nutshell" is the correct expression 🙂  https://dictionary.cambridge.org/dictionary/english/in-a-nutshell